pyrene/src/components/Pill/Pill.tsx

```diff
@@ -50,15 +50,15 @@ const Pill: React.FC<PillProps> = ({
 }: PillProps) => (
   <div>
     {(onClick && icon)
-      && <div styleName={clsx('icon')}><IconButton icon={icon} type={iconType} onClick={onClick} /></div>}
+      && <div className={styles.icon}><IconButton icon={icon} type={iconType} onClick={onClick} /></div>}
     {(!onClick && icon) && (
-      <div styleName={clsx('icon', { [`type-${iconType}`]: true })}>
+      <div className={clsx(styles.icon, { [styles[`type-${iconType}`]]: true })}>
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

`<div className={clsx(styles.icon, styles[`type-${iconType}`] )}>`